frontend/awx/resources/templates/WorkflowVisualizer/types.ts

```diff
@@ -19,14 +21,21 @@ export type LayoutNode = WorkflowNode & {
 };
 
 export interface CustomEdgeProps {
-  element: GraphElement<
-    ElementModel,
+  element: Edge<
+    EdgeModel,
     {
       tag: string;
-      tagStatus: NodeStatus;
-      endTerminalStatus: NodeStatus;
+      tagStatus: NodeStatus | EdgeStatus;
```

Comment:
Since this is an Edge interface, I think we can remove the `NodeState` here, which will allow us to remove some typescript castings

Reply:
good find!